# Request
int64 request_type
string source_node
string destination_node
int64 transaction_id
int64[] numeric_values
string[] string_values
string message

---

# Response
int64 response_type
string return_node
int64 transaction_id
int64[] numeric_values
string[] string_values